pkg/services/store/object/object.proto

```diff
@@ -3,51 +3,65 @@ package object;
 
 option go_package = "./;object";
 
-// The canonical object/document data -- this represents the raw bytes and storage level metadata
-message RawObject {
-  // Unique ID
-  string UID = 1;
+message GRN {
+  // the tenant/org id
+  int64 tenant_id = 1;
+
+  // mabybe "namespace"?  valid values include
+  // * entity
+  // * drive
+  // * service/xyz
+  string scope = 2;
 
   // Identify the object kind.  This kind will be used to apply a schema to the body and
   // will trigger additional indexing behavior.
-  string kind = 2;
+  string kind = 3;
+
+  // Unique ID
+  string UID = 4;
```

Comment:
would it be useful to have an optional `string version = 5;`?

Reply:
It could be... but i'm hoping to keep it as minimal as possible and we can add it if it turns out the UI really needs it.  This will let us have cheaper backing storage if possible (like a file system that does not cheaply know its version).

We do have the etag always though